lib/actions-oclif/internal/osinit.ts

```diff
@@ -0,0 +1,76 @@
+/**
+ * @license
+ * Copyright 2016-2020 Balena Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { stripIndent } from 'common-tags';
+import Command from '../../command';
+import { CommandHelp } from '../../utils/oclif-utils';
+
+// These are internal commands we want to be runnable from the outside.
+// One use-case for this is spawning the minimal operation with root privileges.
+
+interface ArgsDef {
+	image: string;
+	type: string;
+	config: string;
+}
+
+export default class OsinitCmd extends Command {
+	public static description = stripIndent`
+		Do actual init of the device with the preconfigured os image.
```

Comment:
Anyone know if  `actual` has any special meaning here?

Reply:
iirc, this command flashes a SD card (does it really?). I guess what they meant by "actual" is that `balena os initialize` does some "preparations" ahead of the "actual (real) initialization" which is done by the internal `osinit` command.